src/raster/r.colors.out_sld/r.colors.out_sld.py

```diff
@@ -144,8 +144,11 @@ def main():
         # sld+='            <ColorMap type={}>\n'.format('"ramp"')
         ColorMapEntry = '              <ColorMapEntry color="#{0:02x}{1:02x}{2:02x}" quantity="{3}" opacity="{4}" />\n'
 
-    #
+    # loop over colors
+    num = 0
     for c in color_rules:
+        num += 1
+        grass.percent(num, len(color_rules), 1)
```

Comment:
Here is a slightly shorter suggestion:
```suggestion
    for num, c in enumerate(color_rules):
        grass.percent(num + 1, len(color_rules), 1)
```
(not black/flake8 checked...)

Reply:
Thanks, @ninsbl!